src/libraries/System.Drawing.Common/src/Resources/Strings.resx

```diff
@@ -462,6 +462,6 @@
     <value>System.Drawing.Common is not supported on this platform.</value>
   </data>
   <data name="CachedBitmapNotSupported" xml:space="preserve">
-    <value>CachedBitmap is not supported on the current version of libgdiplus.</value>
+    <value>The installed version of libgdiplus is lower than 6.1.</value>
```

Comment:
I'm not good at naming and messages, but I'll use something like
```suggestion
    <value>CachedBitmap is not supported on the current version of libgdiplus. It is supported on version 6.1 onwards.</value>
```
What do you think?

Reply:
Error messages are hard 😛

Reply:
One of the hardest parts in programming 😃